assets/src/components/ladder.tsx

```diff
@@ -158,41 +158,64 @@ const LadderVehicle = ({
   const widthOfVehicleGroup = centerOfVehicleGroupX * 2
   const x =
     vehicleDirection === VehicleDirection.Up ? 47 : -(48 + widthOfVehicleGroup)
-  const y =
-    yForVehicle(vehicle, timepoints, timepointSpacingY, vehicleDirection) || -10
+  const vehicleY =
+    timepointStatusY(
+      vehicle.timepoint_status,
+      timepoints,
+      timepointSpacingY,
+      vehicleDirection
+    ) || -10
+  const roadLineX =
+    vehicleDirection === VehicleDirection.Up ? CENTER_TO_LINE : -CENTER_TO_LINE
+  const scheduledY = timepointStatusY(
+    vehicle.scheduled_timepoint_status,
+    timepoints,
+    timepointSpacingY,
+    vehicleDirection
+  )
   const isSelected = vehicle.id === selectedVehicleId
 
   const rotation = vehicleDirection === VehicleDirection.Down ? 180 : 0
 
   return (
-    <g
-      className="m-ladder__vehicle"
-      transform={`translate(${x},${y}) rotate(${rotation},${centerOfVehicleGroupX},${centerOfVehicleGroupY})`}
-      onClick={() => dispatch(selectVehicle(vehicle.id))}
-    >
-      <VehicleIcon isSelected={isSelected} scale={0.625} />
-      <VehicleLabel vehicle={vehicle} rotation={rotation} x={0} y={26} />
+    <g>
+      {scheduledY && (
+        <ScheduledLine
```

Comment:
The line is still being drawn above previously drawn vehicles. Do we want all lines to be below all vehicles? (@ingridpierre?)

Reply:
Glides has the same bug, and it's barely noticable. I don't think it'd be worth fixing.

Reply:
Followup task: [Draw all schedule lines below all triangles](https://app.asana.com/0/1112935048846093/1126085679577924)

I'm thinking it'll sink to the bottom of design backlog as low priority, but let's collect any screenshots there, and if it ends up being a problem we can bring it back.